services/QuillLMS/client/app/bundles/PremiumHub/containers/__tests__/diagnosticGrowthReports.test.tsx

```diff
@@ -53,7 +54,7 @@ describe('DiagnosticGrowthReports', () => {
       const { asFragment } = render(<DiagnosticGrowthReports {...props} />);
       expect(asFragment()).toMatchSnapshot();
       expect(screen.getByRole('heading', { name: /diagnostic growth report/i })).toBeInTheDocument()
-      expect(screen.getByRole('button', { name: /download/i })).toBeInTheDocument()
+      // expect(screen.getByRole('button', { name: /download/i })).toBeInTheDocument()
```

Comment:
Is this supposed to be cleaned up?

Reply:
I'll add a TODO, this should be added back once the CSV download feature is live